apps/utils.py

```diff
@@ -148,3 +153,54 @@ def send_templated_email(template, subject, context, recipients, sender=None):
     email = EmailMessage(subject, email_body, send_by, recipients)
     email.content_subtype = 'html'
     email.send()
+
+
+class S3PreSignedMixin:
+    def get_content_type(self, extension):
+        mimetypes.add_type('application/vnd.openxmlformats-officedocument.spreadsheetml.sheet', '.xlsx')
```

Comment:
This will append the mimetype mapping every time this function is called - I think if we need to add to the mimetype setting it should be done once - preferably in a settings file.